Makefile

```diff
@@ -69,7 +69,7 @@ gofmt:
 	find . -name '*.go' ! -path './vendor/*' -exec gofmt -s -w {} \+
 	git diff --exit-code
 
-conmon:
+conmon: crio
```

Comment:
I don't think this helps with the “we want to decouple `conmon` from `crio`” use case.  Personally, I'm fine with either:

* The approach taken in #1164, since I don't have @runcom's [aversion to `import "C"` in Go code][1].  But maybe I would if he gave more details on the downside ;).
* Just dropping the constants and passing these in as variables:

    ```console
    $ conmon --buf-size 8192 …
    ```

    That means you need to allocate some variables on the heap instead of [getting them on the stack][1], but I'd rather do that than add `crio` as a build dependency if `crio` and `conmon` were in separate repositories.

[1]: https://github.com/kubernetes-incubator/cri-o/blob/7e5b103236a9e95cbce22bae8ad10a84fdff9c5f/conmon/conmon.c#L623
[1]: https://github.com/kubernetes-incubator/cri-o/pull/1164#discussion_r151844332